CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## [Release 2.12.0](https://github.com/aws-amplify/amplify-android/releases/tag/release_v2.12.0)
+
+### Features
+- **auth:** Removed dependency on oauth2 token for refreshToken and move to initiateAuth for hostedUI ([#2543](https://github.com/aws-amplify/amplify-android/issues/2543))
+
+### Bug Fixes
+- **auth:** Fix for missing exception type for sign in ([#2567](https://github.com/aws-amplify/amplify-android/issues/2567))
+- **datastore:** Prevent stop from interrupting subsequent start call ([#2569](https://github.com/aws-amplify/amplify-android/issues/2569))
+- **datastore:** Ignore ApiAuthExceptions when subscribing or syncing models ([#2561](https://github.com/aws-amplify/amplify-android/issues/2561))
+- add granular read ops enum ([#2265](https://github.com/aws-amplify/amplify-android/issues/2265))
+- **flutter/datastore:** Nullable custom type arrays ([#2565](https://github.com/aws-amplify/amplify-android/issues/2565))
+
+### Miscellaneous
+- fix (push): also catch `RuntimeExecutionException` for google play store device use case ([#2562](https://github.com/aws-amplify/amplify-android/issues/2562))
```

Comment:
This can be moved up. I think the naming was incorrect on this one.

Reply:
yeah that's supposed to go in fixes, is it possible that it doesn't think push is a valid category?  I'd be curious to see what the GH action is doing

Reply:
notification is the category it is expecting I suppose?